src/main/java/io/github/perrymant/moneymaker/Budget.java

```diff
@@ -2,13 +2,14 @@
 
 import com.jakewharton.fliptables.FlipTable;
 
+import java.io.Serializable;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.stream.Collectors;
 
-import static io.github.perrymant.moneymaker.TransactionType.*;
+import static io.github.perrymant.moneymaker.TransactionType.CREDIT;
 
-class Budget {
+class Budget implements Serializable {
```

Comment:
This class didn't have to change at all this sprint. Let's just undo all these changes.